### PR TITLE
refactor(python code): Add a few minor changes

### PR DESCRIPTION
**Pull request by amirmalekian**

### CODE REVIEW

File: python/clog.py [hunk 1]
```diff
@@ -68,8 +65,7 @@ def filter(self, record):
             record.threadName = 0
 
         if "prefix" in record.args:
-            prefix = record.args.get("prefix")
-            if prefix:
+            if prefix := record.args.get("prefix"):
```
Comment (tempookian) [inline python/clog.py:68]: Let's keep compatibility with Python 3.6

**Review by tempookian — CHANGES_REQUESTED** [review]
Just a minor change to keep compatibility

**Review by tempookian — APPROVED** [review]

**Comment by tempookian** [comment]
@MortezaBashsiz as far as Python is concerned, everything is fine and ready to be merged

**Comment by MortezaBashsiz** [comment]
Merged
Thanks

**Pull request merged**